site/content/events/2016-losangeles-1day/_email_organizers.txt

```diff
@@ -1 +1 @@
-organizers-losangeles-2016@devopsdays.org
+devopsdayla-planning@googlegroups.com
```

Comment:
I understand, but we've been running the event with the google group for 6 years now and have our organizers on it as well as our archive history.  It's also been published as part of our call for papers on DevOps.com and a number of other media sites.

If/when  we move to a 2 day event for April, we'll move back to a devopsdays.org address.